### DISCUSSION TITLE
How can I have jitter, bitrate and bytes sent?

### DISCUSSION THREAD

**Discussion by rostamiani** [ObserveRTC/observer#96] [Q&A]
I started `Full stack example` project and saw the reports.
But there is not any report containing jitter, bitrate, bytes sent/received.
The report types are just CALL_EVENT and CALL_META_DATA...

**Comment by balazskreith** [comment]
Give me 1-2 days and take a look on the full-stack-example repo. It might be there is something not passing together. Sink can have. configurations to control what type of reports it can get through and what are not. 
The configuration of a sink is printed out to the console when the observer starts up.
The other explanation that there was no such report for in-, and outbound tracks, because there was not any which was monitored. Have you open two tabs (though one is enough for outbound reports)

**Comment by rostamiani** [comment]
I have OUTBOUND_VIDEO_TRACK report when using google chrome. but not with firefox!

**Reply by balazskreith** [reply]
oh yes! Firefox has problems providing track identifier in stats, https://bugzilla.mozilla.org/show_bug.cgi?id=1804676 discovered it in Dec and reported them through slack. outbound or inbound track reports can only be generated if we have track identifier in some way.
If you use mediasoup in client-monitor-js I added [this](https://github.com/ObserveRTC/client-monitor-js/blob/c14fb550f8544c303198a4e7336896113d92fe01/src/collectors/MediasoupStatsCollector.ts#L37) hack to correct it, and it should automatically turned on if bowser detect firefox.
